scripts/build-macos.sh

```diff
@@ -3,8 +3,6 @@
 if [ "$1" = "all" ]; then
     ./scripts/build-macos.sh cxxlib debug $3
     ./scripts/build-macos.sh cxxlib release $3
-    ./scripts/build-macos.sh python2lib debug $3
```

Comment:
I also removed python V2 compiling from the scripts and CI, since python2 packages are missing in homebrew and saves us a lot of trouble.